frontend/src/scenes/organization/Settings/index.tsx

```diff
@@ -161,6 +165,15 @@ export function OrganizationSettings({ user }: { user: UserType }): JSX.Element
                 <Divider />
                 <Members user={user} />
                 <Divider />
+                {featureFlags[FEATURE_FLAGS.PER_PROJECT_ACCESS] && (
+                    <>
+                        <RestrictedArea
+                            Component={Permissioning}
```

Comment:
I was actually extremely confused when I saw this. I thought you enabled project-based permissioning at the project-level, not org level. That way you can have "default" projects that everyone has access to, and then you can create private projects if you want to. Seems a tad more intuitive. Thoughts?

Reply:
Hmm, I thought we agreed upon this in the approved documentation PR, which outlines the mechanism implemented.
I see your point though, I think both are intuitive but entirely project-based does sound a bit more flexible. I can rework this if you think it's worth it?

Reply:
Sorry I completely misunderstood when reading the docs, got confused on the way we "turn on" project-based permissioning. This will be complex to change in the future so we should ship with whatever makes the most sense, which I'm still unsure about. 

Would love to get @clarkus & @marcushyett-ph (as he's been chatting with customers more frequently on this) input on this too. TLDR; you can now have project-based permissions in addition to org-based permissions. Question is, should we enable this at the org-level (wherein if enabled, every project will be private by default for non-admins) or at the project-level (where in every project will be public to all org members by default unless you turn this setting on for the specific project).

Reply:
I think we need to support both starting points:

e.g. If you've been using PostHog for a while and everyone expects to have access to everything, but you turn on this feature - you'd expect that people can still access everything until specific permissions have been configured

e.g. If they're setting up PostHog for the first time and have no users, they would probably want to grant permissions only to specific people on specific projects.

Reply:
I'm actually a bit against this, I think this will be confusing for users and harder to manage. I believe in any of the options both existing and new users will have the option to treat things as they want.

Reply:
Given these examples, I think option 1 (what Paolo suggests) seems to serve both users really well (I'm also imagining that when creating new projects, they would just have a checkbox "restrict default access"), so I believe it makes sense to implement, though it will make the feature spill over a bit more into the next sprint.

Reply:
I think it's fine to have a bit of spill unto this next sprint as it's very related to what you're working on anyways, but you'll have better context here